### PR TITLE
Support for new endpoints: List Member Activity

### PR DESCRIPTION
**Pull request by p-zielinski**
Why
- customer request

What? 
adding support for
- https://docs.voucherify.io/reference/list-member-activity
- https://docs.voucherify.io/reference/list-member-activity-1

Tests:
Screenshot 2024-10-16 at 16 59 04

### CODE REVIEW

File: packages/sdk/src/types/Loyalties.ts [hunk 1]
```diff
@@ -712,9 +732,11 @@ export interface LoyaltiesGetPointsExpirationResponseBody {
 	total: number
 }
 
-export interface LoyaltiesListCardTransactionsRequestQuery {
```
Comment (p-zielinski) [inline packages/sdk/src/types/Loyalties.ts:715]: duplicated interface

File: packages/sdk/src/types/Loyalties.ts [hunk 2]
```diff
@@ -712,9 +732,11 @@ export interface LoyaltiesGetPointsExpirationResponseBody {
...
-	limit?: number
-	page?: number
+export interface LoyaltiesListCardTransactionsResponseBody {
```
Comment (awilczek) [inline packages/sdk/src/types/Loyalties.ts:735]: but this `LoyaltiesListCardTransactionsResponseBody` is also a duplicate interface now in this PR
the other one has additionally `more_starting_after?: string`

File: packages/sdk/test/loyalties.spec.ts [hunk 3]
```diff
@@ -192,6 +193,12 @@ describe('Loyalties API', () => {
 		} as LoyaltiesListCardTransactionsResponseBody)
 	})
 
+	it('Should list member activity', async () => {
+		const listMemberActivityResponse = await client.loyalties.listMemberActivity(null, loyaltiesMember.code)
+		expect(listMemberActivityResponse.data.length).toBeGreaterThan(0)
+		expect(typeof listMemberActivityResponse.data[0].type).toEqual('string')
```
Comment (awilczek) [inline packages/sdk/test/loyalties.spec.ts:199]: I'm wondering if checking this `type` is enough
there are 4 other properties that could be checked somehow
I know this is not a place for testing precisely everything - we just need to confirm this endpoint works in the SDK, so I suppose this is okey already
but still... maybe checking these 4 other properties (just their types) would make this test a bit better with little effort?

Comment (p-zielinski) [reply packages/sdk/test/loyalties.spec.ts:209]: `data` is tagged as `any`, checking if a 1 property exists is sufficient to know that the rest should be recognized ass well.
Of course in our case this most likely would be a same event oven and over again. 
But what if sometime in the future Voucherify will not create this event but a different one during loyalty card publication. Tests will fail.

**Review by marcinrog — APPROVED** [review]

File: packages/sdk/test/loyalties.spec.ts (hunk 3 above)
Comment (awilczek) [reply packages/sdk/test/loyalties.spec.ts:209]: sure, ok
and thanks for the change, more `expect`s is indeed what I had in mind :)

**Pull request merged**